src/shared/util.js

```diff
@@ -919,7 +928,8 @@ function stringToPDFString(str) {
       try {
         const decoder = new TextDecoder(encoding, { fatal: true });
         const buffer = stringToBytes(str);
-        return decoder.decode(buffer);
+        const decoded = decoder.decode(buffer);
+        return decoded.replaceAll(/\u001b[^\u001b]*(\u001b|$)/g, "");
```

Comment:
Isn't this similar to PR #17165, where we didn't want to run `replaceAll` unconditionally?
```suggestion
        const decoded = decoder.decode(buffer);
        if (!/\x1b/g.test(decoded)) { // Do we want to create this regexp *once*, outside of the function?
          return decoded;
        }
        return decoded.replaceAll(/\x1b[^\x1b]*(\x1b|$)/g, "");
```